wg-agendas/06-18-2019.md

```diff
@@ -0,0 +1,23 @@
+The 2nd GraphiQL Working Group meeting will involve updates on current progress, further discussion of the next major version of GraphiQL in terms of plugin system, desired features, and discussing options in terms of the language server and codemirror/monaco, etc
+
+- **Video Conference Link:** 
+- **Live Notes:**
+- **Date & Time:** May 14th 2019 16:00 - 18:00 UTC
+- **NOTE:** Meeting date and time may change up to a week before the meeting. Please check the agenda the week of the meeting to confirm.
+
+## Attendees
+
+Name                 | Organization         | Location
+-------------------- | -------------------- | ----------------------
+Rikki Schulte        | GraphiQL Maint. Team | Cleveland, OH, US
+*ADD YOUR NAME HERE* | *COMPANY / ORG*      | *WHERE YOU ARE*
+
+## Agenda
+
+1. Introductions
+1. Updates
+  - Access
+  - Licensing
+  - Monorepo
+1. Plugin System
+1. Language Server (LSP?) & Codemirror or Monoco?
```

Comment:
I think it was [Monaco](https://microsoft.github.io/monaco-editor/) 😄

Reply:
nice find, thank you!